yacc/error.c

```diff
@@ -77,7 +77,7 @@ static void print_pos(char *st_line, char *st_cptr)
 }
 
 
-static Noreturn void gen_error(int st_lineno, char *st_line, char *st_cptr, char *msg)
+CAMLnoret static void gen_error(int st_lineno, char *st_line, char *st_cptr, char *msg)
```

Comment:
None of the other functions which end with a `done` have `CAMLnoret` - is it needed at all? (it appears to have been an oversight that `Noreturn` was left used here)

Reply:
It is needed by other functions calling `gen_error` (this is mingw):
```
yacc/error.c: In function 'syntax_error':
yacc/error.c:92:1: error: 'noreturn' function does return [-Werror]
   92 | }
      | ^
```

Reply:
Ah, that makes sense, thanks!